bittorrent/player.go

```diff
@@ -536,6 +536,22 @@ func (btp *BTPlayer) chooseFile() (int, error) {
 			choices = append(choices, candidate)
 		}
 
+		if btp.episode > 0 {
+			foundMatches := int64(0)
+			lastMatched := int(-1)
+			var pattern = fmt.Sprintf(".S%02dE%02d.", btp.season, btp.episode)
+			for index, choice := range choices {
+				if strings.Contains(choice.Filename, pattern) {
+					foundMatches++
+					lastMatched = index
```

Comment:
It's usually better to increase the counter after other operations are done, in this case just setting `lastMatched`, but still, if you can switch those two lines ;)